framework/application/application.h

```diff
@@ -68,18 +73,27 @@ class Application
 
     void SetFileProcessor(decode::FileProcessor* file_processor);
 
+    void HandleMeasurementRange(uint32_t measurement_start_frame,
+                                uint32_t measurement_end_frame,
+                                bool     quit_after_range,
+                                bool     flush_measurement_range);
+
+    void WriteMeasurementRangeFpsToConsole(uint32_t measurement_start_frame, uint32_t measurement_end_frame);
+
   private:
     // clang-format off
-    std::vector<decode::Window*> windows_;          ///< List of windows that have been registered with the application.
-    decode::FileProcessor*       file_processor_;   ///< The FileProcessor object responsible for decoding and processing
-                                                    ///< capture file data.
-    bool                         running_;          ///< Indicates that the application is actively processing system
-                                                    ///< events for playback.
-    bool                         paused_;           ///< Indicates that the playback has been paused.  When paused the
-                                                    ///< application will stop rendering, but will continue processing
-                                                    ///< system events.
-    std::string                  name_;             ///< Application name to display in window title bar.
-    uint32_t                     pause_frame_;      ///< The number for a frame that replay should pause after.
+    std::vector<decode::Window*> windows_;                  ///< List of windows that have been registered with the application.
+    decode::FileProcessor*       file_processor_;           ///< The FileProcessor object responsible for decoding and processing
+                                                            ///< capture file data.
+    bool                         running_;                  ///< Indicates that the application is actively processing system
+                                                            ///< events for playback.
+    bool                         paused_;                   ///< Indicates that the playback has been paused.  When paused the
+                                                            ///< application will stop rendering, but will continue processing
+                                                            ///< system events.
+    std::string                  name_;                     ///< Application name to display in window title bar.
+    uint32_t                     pause_frame_;              ///< The number for a frame that replay should pause after.
+    int64_t                      measurement_start_time;    ///< The time when the measurement range started
```

Comment:
These two should end in "_" to match the style guide.

Reply:
Fixed now!